### PR TITLE
Add default property to EmailDeliverySettings

### PR DESCRIPTION
**Pull request by vova07**

### CODE REVIEW

File: spec/components/schemas/EmailDeliverySettings/EmailDeliverySetting.yaml [hunk 1]
```diff
@@ -41,6 +41,10 @@ properties:
       - postmark
       - sendgrid
     example: "rebilly"
+  default:
```
Comment (ashkarpetin) [inline spec/components/schemas/EmailDeliverySettings/EmailDeliverySetting.yaml]: Shouldn't it be isDefault for boolean?

**Comment by vova07** [comment]
test this please

**Comment by VladimirBerdnik** [comment]
retest this please

**Comment by vova07** [comment]
please review

File: spec/components/schemas/EmailDeliverySettings/EmailDeliverySetting.yaml [hunk 2]
```diff
@@ -41,6 +41,10 @@ properties:
       - postmark
       - sendgrid
     example: "rebilly"
+  isDefault:
+    type: boolean
+    description: Whether this setting will be used by default or not.
```
Comment (adamaltman) [inline spec/components/schemas/EmailDeliverySettings/EmailDeliverySetting.yaml]: This is not a very good description.

**Pull request merged**